okonomiyaki/runtimes/runtime.py

```diff
@@ -98,7 +98,7 @@ def from_prefix_and_platform(cls, prefix, platform, version=None,
             scriptsdir = bindir = posixpath.join(prefix, "bin")
             paths = (bindir, )
 
-        if version.major == 3:
+        if version.major == 3 and platform.os_kind != OSKind.windows:
```

Comment:
If this is used with a non-Enthought Python build, the `python -> python3` link is not guaranteed to exist. In fact, by default, when building Python 3, it does not exist without providing some `configure` options.